lib/printer.rb

```diff
@@ -0,0 +1,18 @@
+require_relative "order"
+
+class Printer
+
+  def format(something_to_print)
+    something_to_print.each { |hash|
+      puts "#{hash[:Dish]}: £#{hash[:Price]}"
+    }
+  end
+
+  def print_bill(complete_order, total_due)
+    complete_order.each { |hash|
```

Comment:
Duplication of the format method here - I think you could have called `format` before then printing the total for the same output